### DISCUSSION TITLE
How to resize the image using image up-loader?

### DISCUSSION THREAD

**Discussion by shanthi-techie** [react-page/react-page#1157] [Q&A]
Is it possible to resize the uploaded image or provide a height and width for an image ?

**Comment by hurricanehunter0702** [comment]
Yes, it is definitely possible to resize an uploaded image or provide a specific height and width for an image in a React application.
There are several libraries. You can try one of these libraries.
react-image-resizer
react-easy-image
react-dropzone
Cheers.
